tracer/test/test-applications/debugger/Samples.Probes/SmokeTests/AsyncRecursiveCall.cs

```diff
@@ -0,0 +1,26 @@
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+
+namespace Samples.Probes.SmokeTests
+{
+    internal class AsyncRecursiveCall : IAsyncRun
+    {
+        [MethodImpl(MethodImplOptions.NoInlining | MethodImplOptions.NoOptimization)]
+        public async Task RunAsync()
+        {
+            await Recursive(0);
+        }
+
+        [MethodImpl(MethodImplOptions.NoInlining | MethodImplOptions.NoOptimization)]
+        [MethodProbeTestData(expectedNumberOfSnapshots:3)]
+        public async Task<int> Recursive(int i)
```

Comment:
This test, like all the others, is a bit problematic in that it's likely that all the code with actually be executed synchronously in practice. Please add `Task.Delay` to force an actual async execution

Reply:
Fixed in 82878f89777debace4f5f6ff669306f87e4c1ae7